.github/workflows/test.yml

```diff
@@ -282,17 +283,19 @@ jobs:
         spec:
           [
             "*.spec.js",
-            "AD integration/**/*.spec.js",
             "admin/**/*.spec.js",
             "checker/**/*.spec.js",
             "dashboard/**/*.spec.js",
             "feedback/**/*.spec.js",
             "login/**/*.spec.js",
             "maker/**/*.spec.js",
+            "maker-about-supply-contract/**/*.spec.js",
+            "maker-bond/**/*.spec.js",
+            "maker-facilities/**/*.spec.js",
+            "maker-loan/**/*.spec.js",
             "number-generator/**/*.spec.js",
             "read-only/**/*.spec.js",
             "reports/**/*.spec.js",
-            "payment-report-officer/**/*.spec.js",
```

Comment:
oh no our file! we'll need to put this back in

Reply:
I wondered why this test wasn't running anymore - thanks for finding it!